scripts/nginx/komga.sh

```diff
@@ -2,7 +2,7 @@
 
 cat > /etc/nginx/apps/komga.conf <<- NGX
 location /komga {
-#   include /etc/nginx/snippets/proxy.conf;
-  proxy_pass        http://127.0.0.1:6800/komga;
+    proxy_pass      http://127.0.0.1:6800/komga;
+    auth_basic      "Realm";
```

Comment:
This stops it from messing up other apps basic auth. Necrod, but if it helps someone in the future:
proxy_set_header Authorization "";
proxy_set_header X-Forwarded-User $remote_user;